README.md

```diff
@@ -1 +1,24 @@
-# Restaurant-Page
\ No newline at end of file
+# Restaurant-Page
+
+## Simple page restaurant for learning setup webpack
+
+### How to run this project
+
+- First: clone the repo and open the folder.
+- Make sure you have npm installed localy then run `npm install` and `npm start`.
+
+### Author
+
+👤 **Ammar El refay Kandel**
+
+- GitHub: [ammarkandel](https://github.com/ammarkandel)
+- Twitter: [@AmmarQandel](https://twitter.com/AmmarQandel)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ammar-kandel-7b4100193/)
+
+### Contributing
+
+This is a project for educational purposes only. We are not accepting contributions.
+
+### Show your support
+
+Give a ⭐️ if you like this project!
```

Comment:
## Improve the README
- Currently, we do have some missing information that seems to be missing and should be added on here in order to have a more professional and well-documented writeup.
- [ ] Do you mind adding a screenshot of the page here?
- [ ] Add more information of what the project is all about here as well.
- [ ] While at it. I would encourage some improvements on the commit messages when conducting these changes. The commit messages should utmost be reflective and informative of what the changes being introduced are about.
- [ ] Do you mind using this [template](https://github.com/microverseinc/readme-template) format for better inspiration in formating?